lib/index.js

```diff
@@ -11,5 +12,6 @@ var main = function (options, callback) {
 
 module.exports = main;
 module.exports.launch = main;
+module.exports.argv = argv;
```

Comment:
Definitely agree, just wanted to say how it currently works.

Launch requires `Config` https://github.com/johanneswuerbach/saucie/blob/master/lib/launch.js#L13, which requires `argv` https://github.com/johanneswuerbach/saucie/blob/master/lib/config.js#L3, which parses the command line parameters automagically :puke: